supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,60 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Benefits of internal linking
+* Users first: Allows users to navigate intuitively.
+* Allows Google to understand architecture of the site.
+* Establishes a quantifiable pathway to recognize topical expertise.
+* They increase site visitation time.
+* Helps Google crawl your site.
+* Builds relevancy and ranking signals for keywords and topics.
+* Builds citation or helps users add context.
+
+.Technical best practices
+* Use standard hyperlinks:
+`<a href="http://www.same-domain.com/" title="Keyword Text">Keyword Text</a>`
+* Do not use non hyperlinks.
+* Do not use JavaScript links. 
+* Do not use Flash Links.
+* Do not use links in Frames or iframes
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Search engines have a soft-cap of crawling around 150 links. Avoid over-linking and consider that the navigational links are crawlable too. 
+
+.Anchor text best practices
+* Contextually describe what the user will find post-click.
+* Contextually describe the content at the link location so that the user can decide if they want to leave their current location. 
+* Avoid irrelevant anchor text.
+* You can use a sentence or sentence fragment as the link anchor text.
+
+
 [[internal-links]]
-== Internal links
+== Example format for internal links
+
+You can use one of the following example link structures for consistency.
+
+.Example AsciiDoc
+----
+For more information about X, see xref/url[Y].
+----
+----
+For more information, see xref/url[Y].
+----
 
 [[external-links]]
-== External links
+== Example format for external links
+
+You can use the following example link structures for consistency.
+
+.Example AsciiDoc
+
+To link to a particular module or section:
+----
+For more information about X, see xref/url[Y], in the _XYZ_ guide.
```

Comment:
Is it link: rather than xref/ ?
Or even xref: ? 
I haven't seen xref/ previously

Reply:
Good point! I'll remove xref. Thank you